tests/fixtures/issues/issue-1453.php

```diff
@@ -1432,59 +1432,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-1453-AB' => [
```

Comment:
This test got moved to the invalid-versions fixture since it's Windows 2000, which as far as I can tell, was never supported by Iron (see the invalid versions fixture for more info).